Gimble_Code/Gimble_Code.ino

```diff
@@ -0,0 +1,178 @@
+#include <Energia.h>
+#include <Servo.h>
+#include "RoveComm.h"
+#include "Gimble_Code.h"
+
+Servo servos[8];
+
+int16_t servoPosition[8]; 
+//uint16_t servoOrigin[8];
+uint16_t servoMax[8];
+uint16_t servoMin[8];
+
+RoveCommEthernet  RoveComm;
+rovecomm_packet packet;
+
+void setup()
+{
+  Serial.begin(115200);
+  
+  delay(ROVECOMM_DELAY);
+  RoveComm.begin(135, 11006);
+  delay(ROVECOMM_DELAY);
+
+  // attaches the servo array to the respective pins (1-8 servos are 0-7)
+  servos[0].attach(SERVO1);
+  servos[1].attach(SERVO2);
+  servos[2].attach(SERVO3);
+  servos[3].attach(SERVO4);
+  servos[4].attach(SERVO5);
+  servos[5].attach(SERVO6);
+  servos[6].attach(SERVO7);
+  servos[7].attach(SERVO8);
+
+  // servo positions are set to starting positions
+  servoPosition[0] = SERVO1_START;
+  servoPosition[1] = SERVO2_START;
+  servoPosition[2] = SERVO3_START;
+  servoPosition[3] = SERVO4_START;
+  servoPosition[4] = SERVO5_START;
+  servoPosition[5] = SERVO6_START;
+  servoPosition[6] = SERVO7_START;
+  servoPosition[7] = SERVO8_START;
+
+  // servo max values put in an array
+  servoMax[0] = SERVO1_MAX;
+  servoMax[1] = SERVO2_MAX;
+  servoMax[2] = SERVO3_MAX;
+  servoMax[3] = SERVO4_MAX;
+  servoMax[4] = SERVO5_MAX;
+  servoMax[5] = SERVO6_MAX;
+  servoMax[6] = SERVO7_MAX;
+  servoMax[7] = SERVO8_MAX;
+
+  // servo min values put in an array
+  servoMin[0] = SERVO1_MIN;
+  servoMin[1] = SERVO2_MIN;
+  servoMin[2] = SERVO3_MIN;
+  servoMin[3] = SERVO4_MIN;
+  servoMin[4] = SERVO5_MIN;
+  servoMin[5] = SERVO6_MIN;
+  servoMin[6] = SERVO7_MIN;
+  servoMin[7] = SERVO8_MIN;
+
+  startupRoutine();
+
+  Serial.println("Gimble Setup Complete");
+}
+
+void loop()
```

Comment:
At some point, current servo positions should likely be sent back to base station for some insight into what this board is thinking.